System.IO.Ports/SerialPort.cs

```diff
@@ -312,7 +314,7 @@ public StopBits StopBits
         /// Gets the port for communications.
         /// </summary>
         /// <remarks>
-        /// .NET nanoFramework doesn't support changing the port.
+        /// DotNET nanoFramework doesn't support changing the port.
```

Comment:
.NET is the correct writing! 😯

Reply:
I know. StyleCop does not like it, but reverted anyway :-D